src/react/backend/location/__tests__/PauseAndResume.test.tsx

```diff
@@ -0,0 +1,211 @@
+import { screen, waitForElementToBeRemoved } from '@testing-library/react';
+import { rest } from 'msw';
+import { setupServer } from 'msw/node';
+import { reduxRender, TEST_API_BASE_URL } from '../../../utils/test';
+import { PauseAndResume } from '../PauseAndResume';
+
+describe('PauseAndResume', () => {
+  const instanceId = 'instanceId';
+  const locationName = 'someLocation';
+  const server = setupServer();
+
+  beforeAll(() => {
+    server.listen({ onUnhandledRequest: 'error' });
+  });
+
+  it('should render the component with pause label when ingestion is enabled', async () => {
+    server.use(
+      rest.get(
+        `${TEST_API_BASE_URL}/api/v1/instance/${instanceId}/status`,
+        (req, res, ctx) =>
+          res(
+            ctx.json({
+              metrics: {
+                ['ingest-schedule']: {
+                  states: { [locationName]: 'enabled' },
+                },
+              },
+              state: null,
+            }),
+          ),
+      ),
+    );
+    reduxRender(<PauseAndResume locationName={locationName} />, {
+      instances: {
+        selectedId: instanceId,
+      },
+    });
+
+    await waitForElementToBeRemoved(() => screen.queryByTestId('loader'), {
+      timeout: 8000,
+    });
```

Comment:
Can we use below option instead? 

```suggestion
    await waitForElementToBeRemoved(() => screen.queryByText('Loading'), {
      timeout: 8000,
    });
```